fastlane/lib/fastlane/actions/hockey.rb

```diff
@@ -172,6 +183,14 @@ def self.author
       def self.is_supported?(platform)
         [:ios, :mac, :android].include? platform
       end
+
+      def self.build_file
+        [
+          @options[:ipa],
+          @options[:apk]
```

Comment:
What if the user provided both values? Maybe we should use the `conflicting_option`

https://github.com/fastlane/fastlane/blob/master/deliver/lib/deliver/options.rb#L36-L40